docs/classad-attributes/job-classad-attributes.rst

```diff
@@ -570,6 +570,11 @@ all attributes.
     in heterogeneous pools, or if the job exited with a signal. Instead,
     see the attributes: :ad-attr:`ExitBySignal`, :ad-attr:`ExitCode`, and
     :ad-attr:`ExitSignal`.
+
+:classad-attribute-def:`FirstStartWaitDuration`
```

Comment:
I see why the attribute is called `FirstStartWaitDuration` however I feel like this could be better unless the RCF's say otherwise. That being said I am fine with this just want to think about it prior to merging

Reply:
Updated with new "batlab tested, RCF-approved naming"!